src/rtree.ml

```diff
@@ -178,6 +178,15 @@ module Make (E : Envelope) (V : Value with type envelope = E.t) = struct
       0 partitions lst
     |> List.map List.rev
     |> List.filter_map (fun v -> if v = [] then None else Some v)
+  
+  (* Rtree Iter Function *)
+  let rec _iter rtree f =
+    match rtree with
+    | Leaf _ -> ()
+    | Node children ->
+      List.iter (fun (_, child) -> _iter child f) children
+    | Empty -> ()
```

Comment:
`f` should also be called on `Empty`.

Reply:
Okay thanks I'll do that

Reply:
Please sir, why do we need to call `f` on `empty`?

Reply:
Good question, the iter function is applying `f` to every value in the tree and `Empty` is one of those just like `Leaf` and `Node`, so it seemed to make sense in my mind to also apply it to `Empty` too :))